shared-helpers/src/locales/general.json

```diff
@@ -189,14 +191,17 @@
   "application.name.dobHelper": "For example: 01 19 2000",
   "application.name.emailPrivacy": "We will only use your email address to contact you about your application.",
   "application.name.firstName": "First Name",
+  "application.name.familyName": "Family name",
+  "application.name.givenName": "Given name",
   "application.name.lastName": "Last Name",
   "application.name.middleName": "Middle Name",
-  "application.name.middleNameOptional": "Middle Name (optional)",
+  "application.name.middleNameOptional": "Middle name (optional)",
```

Comment:
We can't change any of these strings in the English file without also having translations. It's just proven to be too easy to not know that the non-English files need changes still. If we have the new cased versions, we can update them in all languages, but the strings should stay the same with a note/ticket to update them all if we don't have em yet.